tensorflow/contrib/data/python/ops/resampling.py

```diff
@@ -101,11 +101,11 @@ def maybe_warn_on_large_rejection(accept_dist, initial_dist):
                                                    initial_dist_ds))
                           .map(maybe_warn_on_large_rejection))
 
-    current_probabilities_ds = dataset_ops.Dataset.zip(
-        (acceptance_dist_ds, class_values_ds)).map(array_ops.gather)
-    filtered_ds = (
-        dataset_ops.Dataset.zip((class_values_ds, current_probabilities_ds,
-                                 dataset))
+    def _gather_and_copy(class_val, acceptance_prob, data):
+        return (class_val, array_ops.gather(acceptance_prob, class_val), data)
+    current_probabilities_and_class_and_data_ds = dataset_ops.Dataset.zip(
+        (class_values_ds, acceptance_dist_ds, dataset)).map(_gather_and_copy)
+    filtered_ds = (current_probabilities_and_class_and_data_ds\
```

Comment:
you don't need the backslash here, it's within a paren.

bring everything after the paren down one line, indent 4 chars, and indent the .filter below it to start at the same col.

Reply:
Ah this was unintentional. Fixed.